### PR TITLE
(intel-dsa) Migrates Package from BCurran3

### LINKED ISSUE #267: (intel-dsa) Outdated, Incorrect Links, and Not Present In Repository
## Expected Behavior

intel-dsa should be present in this repository, and the package on the CCR should be up to date with valid links to package source etc.

## Current Behavior

The intel-dsa package is outdated, and is still showing links to the previous maintainer's package source and patreon.

## Possible Solution

I feel cheeky even typing out this solution. 😅 

I'd happily submit a PR, but I imagine there's a few things that need to be done to match your standards rather than just pulling the existing code in.

## Steps to Reproduce (for bugs)

See: https://community.chocolatey.org/packages/intel-dsa

## Context

A user has been attempting to get a latest version available on the repository in various ways, and may already have updated code for this.

## Your Environment

N/A

### PR DESCRIPTION
**Pull request by JPRuskin**
## Description
This brings the package code across from BC's repository, and updates it to fit this ownership and repository style.

The update script could probably be trimmed back a little to just contain one of the two methods - parsing the release table is a bit brash when we could just download the binary. Hopefully it's acceptable - and/or it can be trimmed to the repository preference.

## Motivation and Context

The package was claimed by MK, but hasn't been updated or migrated over to this new source location.

Fixes #267 

## How Has this Been Tested?

- Tested updating on Windows 11
- Tested installing and uninstalling on Windows Server 2022

## Types of changes
- ~[ ] Bug fix (non-breaking change which fixes an issue)~
- ~[ ] New feature (non-breaking change which adds functionality)~
- ~[ ] Breaking change (fix or feature that would cause existing functionality to change)~
- [x] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- ~[ ] My change requires a change to documentation (this usually means the notes in the description of a package).~
- ~[ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).~
- [x] I have updated the package description and it is less than 4000 characters.
- [ ] All files are up to date with the latest [Contributing Guidelines](https://github.com/mkevenaar/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Review by mkevenaar — DISMISSED** [review]
Looks good, I am gonna take this update for a spin.

File: automatic/intel-dsa/intel-dsa.nuspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description></description>
+    <summary> Intel® Driver &amp; Support Assistant enables you to scan computing devices for the latest drivers available from Intel.</summary>
+    <releaseNotes>https://www.intel.com/content/www/us/en/support/articles/000056693/software/software-applications.html</releaseNotes>
+    <copyright>Intel Corporation</copyright>
+    <tags>dsa intel driver support assistant update utility</tags>
+    <packageSourceUrl>https://github.com/mkevenaar/chocolatey-packages/tree/master/automatic/intel-dsa</packageSourceUrl>
+    <docsUrl>https://www.intel.com/content/www/us/en/support/topics/idsa-faq.html</docsUrl>
+    <mailingListUrl>https://community.intel.com/</mailingListUrl>
+    <dependencies></dependencies>
```
Comment (JPRuskin) [inline automatic/intel-dsa/intel-dsa.nuspec:22]: May need to add a dependency back in - finally tested on an older system, and got this:

image

That said, any system with recent Chocolatey installed will have a compatible version of dotnet. How far back do you support, with your packages?

```suggestion
    <dependencies>
      <dependency id="dotnetfx" version="4.6.2.0" />
    </dependencies>
```

**Comment by mkevenaar** [comment]
@JPRuskin your changes have been merged, thank you for your contribution!

**Comment by brusezot** [comment]
Hi @mkevenaar, how are you? Are we waiting for the new version? For my part, I've updated the package, so why wait so long?

**Comment by mkevenaar** [comment]
> Hi @mkevenaar, how are you? Are we waiting for the new version? For my part, I've updated the package, so why wait so long?

PR is merged, package is pushed and approved. A new version is already available on the CCR. Nothing else I need to do. If you have an issue with the approved package, please submit a new issue so I can have a look at it.

**Comment by brusezot** [comment]
@mkevenaar thanks for your answere but your title package is not correct : Intel® Driver & Support Assistant (Install) 23.4.39.9, please can you update it : (Intel® Driver & Support Assistant (Install) 24.5.40.11](https://community.chocolatey.org/packages/intel-dsa/24.5.40.11)

**Comment by brusezot** [comment]
Ok, thanks a lot

**Pull request merged**